src/ts/ssh-tcp/services/localPortForwarder.ts

```diff
@@ -105,7 +97,6 @@ export class LocalPortForwarder extends SshService {
 					this.tcpListener2 = await this.pfs.tcpListenerFactory.createTcpListener(
 						listenAddress,
 						this.port,
-						false,
```

Comment:
Please also fix the comment above on line 94: `...and not allowing a port change`

Reply:
Actually this call still should not allow a port change, because it is not expected to listen on different ports between IPv4 and IPv6. I think I'll add the `canChangePort` parameter back to that method.

Reply:
Updated.